claat/parser/gdoc/parse.go

```diff
@@ -650,9 +650,27 @@ func list(ds *docState) types.Node {
 // image creates a new ImageNode out of hn, parsing its src attribute.
 // It returns nil if src is empty.
 // It may also return a YouTubeNode if alt property contains specific substring.
+// or an IframeNode if the alt property contains a URL other than youtube.
 func image(ds *docState) types.Node {
-	if strings.Contains(nodeAttr(ds.cur, "alt"), "youtube.com/watch") {
+	alt := nodeAttr(ds.cur, "alt")
+	if strings.Contains(alt, "youtube.com/watch") {
 		return youtube(ds)
+	} else if strings.Contains(alt, "https://") {
+		u, err := url.Parse(nodeAttr(ds.cur, "alt"))
+		if err != nil {
+			return nil
+		}
+		// For iframe, make sure URL ends in whitelisted domain.
+		ok := false
+		for _, domain := range types.IframeWhitelist {
```

Comment:
It would be cool to display some type of error or warning somehow (not sure how to easily do that) when the domain has not be whitelisted. For instance by adding some Warning text to the "alt" of the image.

If we could do something like `setNodeAttr(ds.cur, "alt", "The domain of the requested iframe <domain here> has not been whitelisted.")` woud work since we are adding the "alt" text to images.

Reply:
good idea, fixed.

Reply:
Cool! Could you also print a warning? it will make it easier to debug and maybe easier for external people that runs claat standalone. :)

Reply:
Great idea, added, and thanks for the presumptive approval. Will merge after build passes travis.